src/engine/meta/meta_blks_mgr.cpp

```diff
@@ -412,6 +423,7 @@ void MetaBlkMgr::add_sub_sb(const meta_sub_type type, const void* context_data,
 
     // add meta_bid to in-memory for reverse mapping;
     m_sub_info[type].meta_bids.insert(meta_bid.to_integer());
+    create_sobject(type);
```

Comment:
Have you verified that multiple call to create_object to with same type and name is fine in sisl sobject framework? 
Even though framework is tolerant to that, I don't think it is good to rely on that, reason being if in the future framework changed and won't support that, this code shouldn't need any change, agree?

Reply:
Agree. The check is added in `create_sobject`